contents/handbook/growth/sales/lead-scoring.md

```diff
@@ -0,0 +1,42 @@
+| Employee Count | 1-10                                                                                                               | 0     |
+|                | 11-1000                                                                                                            | 10    |
+|                | 1000+                                                                                                              | 20    |
+| Ability to pay | Estimated Revenue $0m-$1m                                                                                          | 0     |
+|                | Estimated Revenue $1m-$10m                                                                                         | 5     |
+|                | Estimated Revenue $10m-$100m                                                                                       | 10    |
+|                | Estimated Revenue $100m+                                                                                           | 20    |
+| Role           | engineering                                                                                                        | 10    |
+|                | product                                                                                                            | 10    |
+|                | leadership/founder                                                                                                 | 10    |
+|                | marketing                                                                                                          | 5     |
+|                | other                                                                                                              | 0     |
+| Sub-role       | data_science_engineer                                                                                              | 10    |
+|                | project_engineer                                                                                                   | 10    |
+|                | software_engineer                                                                                                  | 10    |
+|                | web_engineer                                                                                                       | 10    |
+|                | founder/ceo                                                                                                        | 10    |                 
+|                | other                                                                                                              | 0     |
+| Country        | Austria, Canada, France, Germany, Japan, Norway, Sweden, USA                                                       | 10    |
+|                | Australia, Belgium, Estonia, Finland, Georgia, Guernsey, Netherlands, New Zealand, Poland, Portugal, Singapore, UK | 5     |
```

Comment:
It's weird to me that UK companies (given how well funded startups are here) are weighted lower than France, Germany etc. I wonder if we should still call it 10 given that's it's probably just a statistical quirk from our history that they pay less?

Reply:
updated UK score to 10 ✅